bootloader/bootloader.go

```diff
@@ -372,16 +372,16 @@ func ForGadget(gadgetDir, rootDir string, opts *Options) (Bootloader, error) {
 	return nil, ErrBootloader
 }
 
-// BootFile represents each file in the chains of trusted assets and kernels
-// used in the boot process. A boot file can be an EFI binary or a snap file
-// containing an EFI binary.
+// BootFile represents each file in the chains of trusted assets and
+// kernels used in the boot process. For example a boot file can be an
+// EFI binary or a snap file containing an EFI binary.
 type BootFile struct {
 	// Path is the path to the file in the filesystem or, if Snap
 	// is set, the relative path inside the snap file.
 	Path string
 	// Snap contains the path to the snap file if a snap file is used.
 	Snap string
-	// Role is set to the role of the bootloader this boot image
+	// Role is set to the role of the bootloader this boot file
```

Comment:
minor nitpick^100000, but shouldn't doc-comments end in "." ?

Reply:
there's one line more to this below:

// originates from.

otherwise this wouldn't just miss a period but would be fairly ungrammatical.

Reply:
ah yes sorry I missed this in the diff